### DISCUSSION TITLE
Custom settings for GitHub client

### DISCUSSION THREAD

**Discussion by DarkteK** [community/community#24089] [Apps, API and Webhooks]
<p>I added this parameters to my <code>.gitconfig</code> file with the following config:</p>
<pre><code class="lang-auto">[core]
        autocrlf = input
        safecrlf = false
</code></pre>
<p>My question is: Can I add that same configuration to the Github client config?</p>

**Comment by steveward** [comment, marked as answer]
<p>Welcome <a class="mention" href="/darktek">@DarkteK</a>! GitHub Desktop will respect any configuration options you set in your <code>.gitconfig</code>. Let me know if you have any other questions about this.</p>
